code-injection-analysis/InitVigilance_jamieDSOTesting/CWE_079_094_116.js

```
@@ -0,0 +1,161 @@
+// should trigger codeql/javascript/ql/src/Security/CWE-079/UnsafeJQueryPlugin.ql
+
+jQuery.fn.copyText = function(options) {
+	// BAD may evaluate `options.sourceSelector` as HTML
+	var source = jQuery(options.sourceSelector),
+	    text = source.text();
+	jQuery(this).text(text);
+}
+
+// should trigger codeql/javascript/ql/src/Security/CWE-079/Xss.ql
+
+function setLanguageOptions() {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'setLanguageOptions'. Consider to delete it or rename it.

'setLanguageOptions' is defined here:


(lint/suspicious/noRedeclare)